src/Engine/Components/Control/EngineController.cpp

```diff
@@ -127,6 +127,8 @@ void EngineController::goToMainMenu() {
             throw Exception("Couldn't return to main menu");
     };
 
+    closeOverlays();
```

Comment:
I think we can just do `engine->disableOverlays()` here, w/o intermediate functions?